Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -688,10 +688,12 @@ bool EventHandler::focus_previous_element()
     return element;
 }
 
-constexpr bool should_ignore_keydown_event(u32 code_point)
+constexpr bool should_ignore_keydown_event(u32 code_point, u32 modifiers)
 {
+    if (modifiers & (KeyModifier::Mod_Ctrl | KeyModifier::Mod_Alt))
+        return true;
```

Comment:
Yeah this can get annoying :)
But let's add a FIXME to this early return to handle things like ctrl+c & ctrl+v?

Reply:
I think those key combinations would be handled before this function is called, right? Should I put some FIXMEs in the appropriate places before this function is called?

Reply:
Hmm yeah probably handled before this function. I didn't realize there was already a FIXME below, so let's not remove that (it's certainly not resolved), and instead just move it to the calling code.

Reply:
I've added some FIXMEs about handling shortcut keys.